frontend/package.json

```diff
@@ -3,36 +3,56 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/jest": "^27.5.2",
-    "@types/node": "^16.18.36",
-    "@types/react": "^18.2.13",
-    "@types/react-dom": "^18.2.6",
+    "@emotion/react": "^11.10.8",
+    "@emotion/styled": "^11.10.8",
+    "@headlessui/react": "^1.7.14",
+    "@heroicons/react": "^2.0.18",
+    "@material-ui/core": "^4.12.4",
+    "@mui/material": "^5.12.2",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^13.0.0",
+    "@testing-library/user-event": "^13.2.1",
+    "@types/jest": "^27.0.1",
+    "@types/node": "^16.7.13",
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "bn.js": "^5.2.1",
+    "buffer": "^6.0.3",
+    "classnames": "^2.3.2",
+    "nanoid": "^4.0.0",
+    "nft.storage": "^7.0.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.11.1",
+    "react-twitter-embed": "^4.0.4",
+    "typescript": "^5.0.4",
+    "useink": "^1.4.4",
```

Comment:
The latest version is `1.8.0`. I updated the README for `useink` that has a link to the latest example repo. It changed to [this](https://github.com/paritytech/useink/tree/main/playground) (in the useink repo).